style.css

```diff
@@ -0,0 +1,3 @@
+.main-header {
+  color: #4853e4;
+}
```

Comment:
- [ ] It is required to use the class name given to your `h1` tag inside your index.html file to style the content of that tag. Kindly use that class name selector to reference that tag.